sdks/python/apache_beam/runners/pipeline_context.py

```diff
@@ -310,8 +310,9 @@ def get_or_create_environment_with_resource_hints(
       """Creates an environment that has necessary hints and returns its id."""
       template_env = self.environments.get_proto_from_id(template_env_id)
       cloned_env = beam_runner_api_pb2.Environment()
+      # (TODO https://github.com/apache/beam/issues/25615)
       # Remove the suppress warning for type once mypy is updated to
-      # newer version. https://github.com/apache/beam/issues/25615
```

Comment:
Can we say: 
      # Remove the suppress warning for type once mypy is updated to  0.941 or higher?